clang/lib/AST/ByteCode/Compiler.cpp

```diff
@@ -15,6 +15,7 @@
 #include "PrimType.h"
 #include "Program.h"
 #include "clang/AST/Attr.h"
+#include "clang/AST/OperationKinds.h"
```

Comment:
What is this needed for? A call to `isLogicalOp()` was already in the file before.

Reply:
Hmm, I think it introduced by clangd plugin's IWYU feature, I'll revert this change.